index.html

```diff
@@ -26,22 +28,151 @@
-                   <li><a href="#"><img class ="medium" src="PImages/Vector5.png" alt ="Medium Icon"></a></li>
+                   <li><a href="https://github.com/Zuwairanajma"><img class= "github" src="PImages/GitHub.png" alt="Github Icon"></a></li>
+                   <li><a href="https://www.linkedin.com/in/zuwaira-sadiq-566b891b0?"><img class= "linkedin"src="PImages/Linkedin.png" alt="Linkedin Icon"></a></li> 
+                   <li><a href="#"><img class= "angeli"src="PImages/VectorH.png" alt="Angellist Icon"></a> </li>
+                   <li><a href="https://twitter.com/@SadiqJuwairiyya"><img class= "twitter" src="PImages/Twitter.png" alt="Twitter Icon"></a></li>
+                   <li><a href="#"><img class= "medium" src="PImages/Medium.png" alt="Medium Icon"></a></li>
+
                 </ul>
             </nav>
         </div>
     </section>
     <!-- Headline Section End -->
+  
+    <!-- Recent Works Section Start -->
+    <section id="My-Recent" class="My-Recent">
+        <div id="recent-works_container" class="recent-works">
+        <h1 id="my-work" class="my-work">My Recent Works</h1>
+      </div>
+
+        <div>
+        <img class = "placeholder" src="PImages/Img Placeholder.png" alt=""></div>
+            <div id="multip" class="multipost">
+                <h2 id="multi" class="multi">Multi-Post Stories</h2>
+                <p id="stories" class="stories">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </div>
+            <div id="buttons1" class="buttonss1">
+                <button type="button" class="btn btn-primary btn-lg">CSS</button>
+                <button type="button" class="btn btn-primary btn-lg">HTML</button>
+                <button type="button" class="btn btn-primary btn-lg">BootStrap</button>
+                <button type="button" class="btn btn-primary btn-lg">Ruby</button>
+            </div>
+            <buttons id="seeproject" class="btn2 btn-primary btn-lg">See Project</buttons>
+    </section>
+    <section class="cardworks" id="cards">
+      <ul class = cardsarr>  
+        <li class = "cardlist"><div id="cwaction1" class="cwaction1">
+            <h2 class="card1">Professional Art Printing Data</h2>
+            <p class="cardtext">Profesional Art Printing DataA daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <button type="button" class="btn3 btn-primary btn-lg">HTML</button>
+              <button type="button" class="btn3 btn-primary btn-lg">BootStrap</button>
+                <button type="button" class="btn3 btn-primary btn-lg">Ruby</button>
+                <div class ="footer"><button class = "projects btn-primary btn-lg">See project</button></div>
```

Comment:
- [ ] Kindly consider applying display: flex in each card as per requirements. 
- [ ] Also consider taking these technologies in a `ul`. HTML, Bootstrap, and Ruby, each will be an individual `li` to make your code following best practices.
  - You may apply these requirements in all the project cards.